cohortextractor/process_covariate_definitions.py

```diff
@@ -16,7 +16,9 @@ def process_covariate_definitions(covariate_definitions):
     covariate_definitions = apply_compatibility_fixes_for_include_date(
         covariate_definitions
     )
-    covariate_definitions = add_include_date_flags_to_columns(covariate_definitions)
+    covariate_definitions = ensure_source_columns_genereate_required_values(
```

Comment:
Only looking at this, because I looked at https://github.com/opensafely/documentation/pull/422/ as I watch that repository, and it referenced this PR here.

Guessing there's no intentional reason behind the misspelling of `generate`?

Reply:
Thanks @StevenMaude, my vim autocompletion has a habit of propagating typos once they're made!